### ISSUE TITLE
Patched CKE Branch

### ISSUE THREAD

**Issue by tablatronix** [GetSimpleCMS/GetSimpleCMS#743]
:information_source: **there is now a plugin that accomplishes this, but only for the content editor, http://get-simple.info/extend/plugin/gsckepatch/899/** it may be easier for you to install plugins than a sepecial branch, it is also reversable and safer if you are not familiar with installing GS

Stable patched to CKEditor NEW
**GS v3.3.4 w CKE @ 4.3.2**

:arrow_down:   https://github.com/GetSimpleCMS/GetSimpleCMS/archive/stable_cke_4.3.2.zip
- merged in stable v3.3.4
- updated to cke 4.3.2 7b0ba47dc2084ee9568fdf06c3bfccff111fb538
- updated codemirror plugin from codemirror svn ( fatal fixes )

**Comment by PrplHaz4** [comment]
Working with IE11 and Chrome 40, on a HiDPI screen and the toolbar icons are incorrect - some of them are partially cut off and tooltips are not consistent with the icon.  I have a customer with a normal DPI screen on IE11 reporting the same problem.

![image](https://cloud.githubusercontent.com/assets/841414/6042960/05ac19fa-ac57-11e4-8634-4e1a63792f32.png)

**Comment by tablatronix** [comment]
ckeditor latest is 4.4.7, maybe i should bump this up.

have you tried the cked path plugin , it is newer
http://get-simple.info/extend/plugin/gsckepatch/899/

**Comment by PrplHaz4** [comment]
@tablatronix  :+1: the ckepatch plugin fixed the toolbar for me on both IE and Chrome...looks like it unfortunately "undoes" the codemirror source editor...oh well!  Thanks, again!

**Comment by tablatronix** [comment]
What codemirror source editor? Was it something you added yourself ?

**Comment by PrplHaz4** [comment]
no, definitely not - i may be mistaken, but when i initially installed the
CKE patched distributable (from the downloads page) the source editor was
different (it highlighted the tags) from what it is now - basically just
plain text view of source....

**Comment by tablatronix** [comment]
I might have bundled the codemirror plugin in it, let me check.

**Comment by PrplHaz4** [comment]
@tablatronix just checked my download and it it definitely does include codemirror (also just realizing that your original post in this issue also says so!)

**Comment by tablatronix** [comment]
hehe, I think i left it out of the plugin since it is not present in 3.3.x
cke_patch branch was based off of 3.3.4

I also had some problems with the latest version and the plugin, plugins sometimes lag behind cke.
